packages/zoe/tools/internal-types.js

```diff
@@ -1,6 +1,9 @@
 /**
  * @typedef {object} ManualTimerAdmin
- * @property {(msg?: string) => Promise<void>} tick
+ * @property {(msg?: string) => ERef<void>} tick Advance the timer by one tick.
+ * DEPRECATED: use `await tickN(1)` instead.  `tick` function errors might be
+ * thrown synchronously, even though success is signaled with `void` or
+ * `Promise<void>`.
```

Comment:
```suggestion
 * @deprecated Use `await tickN(1)` instead. `tick` function errors might be
 * thrown synchronously, even though success is signaled by returning anything
 * other than a rejected promise.
 * @property {(msg?: string) => void | Promise<void>} tick Advance the timer by one tick.
```

Reply:
Unfortunately, `@deprecated` only displays its doc string for a declaration it decorates, not a `@property`.